src/app/components/ProfileCard/ProfileCard.component.html

```diff
@@ -0,0 +1,13 @@
+<div
+    *ngFor="let item of profileData"
+    class="profile-card"
+>
+    <img
+        [src]="imageUrl|| item.profilePicture"
```

Comment:
Use a Nullish coalescing operator (??) instead. this purpose checks undefined and null in this use case.

Reply:
The left side of this nullish coalescing operation does not include 'null' or 'undefined' in its type, therefore the '??' operator can be safely removed. This is from sonar lint, think it makes sense